kubecost/templates/cluster-controller/cluster-controller-cluster-role.yaml

```diff
@@ -64,4 +64,9 @@ rules:
     resources: ['*']
     verbs: ['list', 'get', 'delete']
   {{- end }}
+  {{- if .Values.clusterController.rbac.resourceQuotaRightSizing }}
+  - apiGroups: [ '' ]
+    resources: [ 'resourcequotas' ]
+    verbs: [ 'get', 'create', 'patch' ]
```

Comment:
Just curious. Do you think you'll need 'list'? It seems like a commonly used verb.